projects/sandbox/src/app/debias/debias.component.html

```diff
@@ -17,6 +17,14 @@
 
     <a class="open-info" (click)="toggleDebiasInfo($event)" title="Debias Tool Information"></a>
 
+    <a
```

Comment:
when I click this, it downloads two times the file.
![afbeelding](https://github.com/user-attachments/assets/dd938e78-397c-46f8-8dc7-98c2c687a7b3)

Reply:
I see - fixed!